Userland/Shell/Shell.cpp

```diff
@@ -1079,17 +1079,18 @@ void Shell::block_on_job(RefPtr<Job> job)
         }
     } };
 
-    Core::EventLoop loop;
+    bool job_exited { false };
     job->on_exit = [&, old_exit = move(job->on_exit)](auto job) {
         if (old_exit)
             old_exit(job);
-        loop.quit(0);
+        job_exited = true;
     };
 
     if (job->exited())
         return;
 
-    loop.exec();
+    while (!job_exited)
+        Core::EventLoop::current().yield();
```

Comment:
`pump()` would be a better fit here IMO. "Pumping the event loop" is a well-understood concept, but "yielding the event loop" is not (at least to me :)

Reply:
Ah, if that's already a known good term here, I'll use that.
the comment above `pump()` really threw me off, it seems